configs/ci/permissions/subscriptions.json

```diff
@@ -12,6 +12,14 @@
             "verb": "write"
         }
     ],
+    "organization": [
+        {
+            "verb": "read"
+        },
+        {
+            "verb": "manage"
```

Comment:
After meeting with the platform team, we have decided to keep using the `write` permission to be consistent with the rest of the platform.

They are planning to do some work in the future to allow configuring implied permissions, which should help act as a guard rail for users creating their own custom roles.

So, if we just `s/manage/write` for these new permissions, we should be in good shape.